tools/docker/core/Dockerfile

```diff
@@ -14,16 +14,11 @@ WORKDIR /opt/certbot
 
 # Copy certbot code
 COPY CHANGELOG.md README.rst src/
-COPY letsencrypt-auto-source/pieces/dependency-requirements.txt .
-COPY letsencrypt-auto-source/pieces/pipstrap.py .
+COPY letsencrypt-auto-source/pieces/dependency-requirements.txt letsencrypt-auto-source/pieces/
```

Comment:
I think we are only copying `letsencrypt-auto-source/pieces/dependency-requirements.txt` here, but we're copying it into a directory named `letsencrypt-auto-source/pieces/` under the current working directory of the `Dockerfile`. We have to put it in a directory with that name because that's where `tools/pip_install.py` expects to find it and this PR modifies our DNS plugin Docker images to use that script.

I think this is confusing though so I'll add a comment here tomorrow explaining what's going on.

Reply:
Ah yes ! I was not careful enough because I was looking at the destination path. Indeed I try to follow the convention of always specifying the complete destination path and not only the destination directory in COPY.

That makes always clear what is copied.

So, as you wish, but having the complete path on dest file will be sufficient in my opinion.